lib/ibmdb.js

```diff
@@ -223,10 +230,18 @@ IBMDB.prototype.executeSQL = function(sql, params, options, callback) {
     }
   }
 
-  conn.query(sql, params, function(err, data, more) {
+  // Build the stmt object that will be passed into the query call.
+  // This is done because the query call can take an object or a set
+  // of parameters.  Depending on the SQL being passed in the call with
+  // parameters may fail due to improper handling in the lower levels.
```

Comment:
We should also move this comment directly above the function you're commenting on. I think it's on 241.